packages/mephisto-worker-experience/src/Functions.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Creates a tip in the format that is accepted
+ * by the handleMetadataSubmit function from the
+ * mephisto-task library
+ * @param {string} header
+ * @param {string} text
+ */
+export function createTip(header, text) {
+  if (!header || !typeof header === "string" || !header instanceof String) {
+    return new Promise(function (resolve, reject) {
+      reject("Tip header is not a string");
+    });
+  }
+  if (!text || !typeof text === "string" || !text instanceof String) {
+    return new Promise(function (resolve, reject) {
+      reject("Tip text is not a string");
+    });
+  }
```

Comment:
curious... what's the benefit of using a Promise here for validation/error handling? I'm a little unfamiliar with this pattern. Have you seen it used in other libraries?

It makes the function return signature is a little weird: `Promise | { header, text, type }`

Reply:
I thought there was some value in this because you could attach a .catch to the createTip() and do something with it for error handling. Not sure if this is too needed though because the next pr makes createTip a function that is not exported publicly.